docs/shrinkwrap.md

```diff
@@ -32,12 +32,13 @@ important that your node_modules is deleted before you do this to be sure to pic
 
 - Install [shonkwrap](https://github.com/skybet/shonkwrap) globally: `npm install -g shonkwrap`. This package attempts
 to remove the from/resolved fields if possible.
+- (Optional) Modify package.json. For example: `npm install --save lodash@4.11.1` or `npm uninstall --save left-pad`
 - Run `make distclean` to delete local node_modules
 - Delete your local copy of npm-shrinkwrap.json.
 - Run `npm install`
 - Verify that Calypso works as expected and that tests pass.
 - Run `shonkwrap --dev`
-- Commit the new npm-shrinkwrap.json
+- Commit the new npm-shrinkwrap.json and changes to package.json.
```

Comment:
Since `package.json` changes are optional, might want the text here to reflect that in case someone didn't make changes and is confused why they'd need to commit the file. 

```text
Commit the new npm-shrinkwrap.json and any changes to package.json
```

```
Commit the new npm-shrinkwrap.json and changes to package.json, if any were made.
```